tests/scripts/generate_test_keys.py

```diff
@@ -89,6 +89,32 @@ def get_look_up_table_entry(key_type: str, group_id_or_keybits: str,
     yield "      {0}, sizeof({0}),\n".format(priv_array_name)
     yield "      {0}, sizeof({0}) }},".format(pub_array_name)
 
+
+def write_output_file(output_file_name: str, arrays: List[str], look_up_table: List[str]):
+    with open(output_file_name, 'wt') as output:
+        output.write("""
+/*********************************************************************************
+ * This file was automatically generated from tests/scripts/generate_test_keys.py.
+ * Please do not edit it manually.
+ *********************************************************************************/
+""")
+        output.write(''.join(arrays))
+        output.write("""
+struct predefined_key_element {{
+    int group_id;  // EC group ID; 0 for RSA keys
+    int keybits;  // bits size of RSA key; 0 for EC keys
+    const unsigned char *priv_key;
+    size_t priv_key_len;
+    const unsigned char *pub_key;
+    size_t pub_key_len;
+}};
+
+struct predefined_key_element predefined_keys[] = {{
+{}
+}};
+""".format("\n".join(look_up_table)))
+
+
 def main() -> None:
     #pylint: disable=too-many-locals
```

Comment:
Question: can we remove this line now that writing to the output file has been moved to a separate function? (No problem if we can't, just checking.)

Reply:
Ops sorry, I was sure I added this change in https://github.com/Mbed-TLS/mbedtls/pull/9017/commits/2c5899664e0d38076f18471bcda1219c5141a800 (even that commit's message reports this), but apparently I missed that somewhere. Anyway I updated that commit (new version is https://github.com/Mbed-TLS/mbedtls/pull/9017/commits/cc403cb6ec2a8ad4aa8ef02efa753d1d643e0df4) so this should be fine now